renovate.json

```diff
@@ -6,7 +6,8 @@
     "dev"
   ],
   "ignoreDeps": [
-    "core-js"
+    "core-js",
+    "fs-extra"
```

Comment:
@ricardogobbosouza Could you please add a todo comment here for tracking the real root issue ?

Reply:
Sure!